src/index.html

```diff
@@ -22,6 +22,353 @@
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <button class="card__button">buy</button>
+      </div>
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews-value">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <button class="card__button">buy</button>
+      </div>
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews-value">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <button class="card__button">buy</button>
+      </div>
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews-value">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <button class="card__button">buy</button>
+      </div>
```

Comment:
You're repeating the same structure for the card multiple times. In a real-world scenario, this should be done dynamically using JavaScript or a server-side language to loop over your data and generate the HTML for each card. This will make your code cleaner and more maintainable.